scania-digital-design-system/sdds
# fix(icons): enforce specific version of @scania/icons

**Describe pull-request**  
Due to issue of @scania/icons version 1.1.2 enforcing version 1.1.1 shows icons in the sdds-icon component again.

**How to test**  
1. Go to preview link below
2. Check in Components -> Icons -> Component
3. Make sure icons are showing
